compliance_checker/suite.py

```diff
@@ -723,7 +723,13 @@ def load_remote_dataset(self, ds_str):
         :param str ds_str: URL to the remote resource
         '''
 
-        if opendap.is_opendap(ds_str):
+        if erddap.is_tabledap(ds_str):
+            return Dataset(
+                ds_str,
+                mode="r",
+                memory=erddap.get_tabledap_bytes(ds_str, "ncCF").getbuffer()
+            )
+        elif opendap.is_opendap(ds_str):
```

Comment:
Consider adding a case for files that have a mimetype of `application/x-netcdf` so that you can hit a netCDF file directly.  Below is an example that only grabs part of the data.

E.g.:
```
curl -I 'https://gliders.ioos.us/erddap/tabledap/amelia-20180501T0000.ncCF?&time%3E=max(time)-1%20day'
HTTP/1.1 200 
Server: nginx/1.16.1
Date: Fri, 24 Apr 2020 19:36:26 GMT
Content-Type: application/x-netcdf
Connection: keep-alive
Last-Modified: Fri, 24 Apr 2020 19:36:26 GMT
xdods-server: dods/3.7
erddap-server: 2.02
Content-Disposition: attachment;filename=amelia-20180501T0000_dec7_6846_3101.nc
Content-Encoding: identity
Access-Control-Allow-Origin: *
```

Reply:
Resolved by #801